src/argilla/server/schemas/v1/common.py

```diff
@@ -0,0 +1,27 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import List
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class BulkDelete(BaseModel):
+    ids: List[UUID]
+
+
+class BulkDeleteResponse(BaseModel):
```

Comment:
Related to my previous comment. The happy path for this endpoint can be a 204 response. In that case, this schema is not needed anymore.